app/homepage.py

```diff
@@ -57,6 +57,60 @@ def choose_map_style(n_clicks):
     return button_content_map, map_object, slider
 
 
+#This function returns the user selection area in the left side bar
+def user_selection_area():
+    """ This function returns the basic dash component of the left side bar
+    of the monitoring platform and has been created because it is used several time within the app.
+
+    Parameters:
+    None
+
+    Returns:
+    html dash components: A set of components that composed the left side bar of the app
```

Comment:
I know it does not add anything to your code but it's a 'list' of components and not a 'set' :)